### ISSUE TITLE
Crash on Version 5.0

### ISSUE THREAD

**Issue by BearsMan** [HasKha/GWToolboxpp#596]
My Crash from I tested is only when I rezone to a new explorable area such as doing DoA or SoOSC 
[5.0-20210512-061804-22204-20668.dmp.zip](https://github.com/HasKha/GWToolboxpp/files/6465485/5.0-20210512-061804-22204-20668.dmp.zip)
I finally have me a crash dump

**Comment by 3vcloud** [comment]
Can you try disabling the friend list module and seeing if this fixes the issue please? If you get another crash, please attach it

**Comment by BearsMan** [comment]
> Can you try disabling the friend list module and seeing if this fixes the issue please? If you get another crash, please attach it

I did not get a TB Crash instead it was a game client crash

**Comment by BearsMan** [comment]
[5.0-20210512-084627-19756-23376.dmp.zip](https://github.com/HasKha/GWToolboxpp/files/6466217/5.0-20210512-084627-19756-23376.dmp.zip)

@3vcloud This is the dump for crash with Friends List Module Turned Off

**Comment by 3vcloud** [comment]
Nope, looks like the friend list module is still enabled and is being shown on-screen - can you double check that its off then try again please?

**Comment by BearsMan** [comment]
@3vcloud I checked with the module off and it was only a client crash not a TB crash

**Comment by BearsMan** [comment]
![cZDDfxDin5](https://user-images.githubusercontent.com/34116754/118030432-49ee7380-b333-11eb-8303-57430c95712f.jpg)
Recent Crash with Toolbox Enabled and Friends List Module Disabled

**Comment by omelkonian** [comment]
I also got a crash:
```
*--> Crash <--*
Assertion: Validate(id)
p:\code\engine\frame\frmsg.cpp(597)
App: Gw.exe
BaseAddr: 00A40000
ProgramId: 1
Build: 37121
When: 5/12/2021 20:02:10
Flags: 0

*--> System <--*
Name: DESKTOP-04G5QE5
IpAddr: 192.168.1.7
Processors: 8 [GenuineIntel:6:14:9]
OSVersion: 10.0 (64 bit)

*--> Thread 0xfffffffe <--*
eax=00000000 ebx=7691e7b0 ecx=00000000 edx=00000000 esi=039cf274 edi=00fe10bc
eip=00ac267b esp=039cf268 ebp=039cf284
cs=0202 ss=0023 ds=002b es=002b fs=002b gs=0053 efl=0000002b

ebx-32 7691E790  8b55ff8b 25ff5dec 76981188 cccccccc 
ebx-16 7691E7A0  cccccccc cccccccc cccccccc cccccccc 
ebx +0 7691E7B0  c358fe6a cccccccc cccccccc cccccccc 
ebx+16 7691E7C0  8b55ff8b 358b56ec 769b08dc 850ff685 
ebx+32 7691E7D0  00010079 5d5ec033 cc000cc2 cccccccc 
ebx+48 7691E7E0  8b55ff8b 084d8bec f040c033 4001c10f 
esi-32 039CF254  00ac26b0 00fe10bc 00fe0f24 00000255 
esi-16 039CF264  039cf274 00fe10bc 00fe0f24 00fe0f24 
esi +0 039CF274  00ac267b 039cf268 039cf284 6fe41653 
esi+16 039CF284  039cf294 00c4105e 00000255 327882f8 
esi+32 039CF294  039cf2a0 00c2cbb0 000000b7 039cf2e8 
esi+48 039CF2A4  00b68ab0 000000b7 00000000 327882f8 
esi-32 00FE109C  736d3e2d 5f6d2e67 73616c63 2e736573 
esi-16 00FE10AC  6e756f43 00292874 636f7270 00000000 
esi +0 00FE10BC  696c6156 65746164 29646928 00000000 
esi+16 00FE10CC  635c3a70 5c65646f 69676e65 665c656e 
esi+32 00FE10DC  656d6172 7272665c 2e746365 00707063 
esi+48 00FE10EC  3faaaaab 635c3a70 5c65646f 69676e65 

*--> Code <--*
00AC265B  cccccccc cc558bec 83ec1ca1 54b10501 .....U......T...
00AC266B  33c58945 fc8955ec 894de4e8 00000000 3..E..U..M......
00AC267B  8f45f054 8f45f455 8f45f883 7dec0074 .E.T.E.U.E..}..t
00AC268B  088b45ec 8945e8eb 07c745e8 dc50ed00 ..E..E....E..P..
00AC269B  8d4df051 8b550852 8b45e850 8b4de451 .M.Q.U.R.E.P.M.Q
00AC26AB  e8900000 0083c410 8b4dfc33 cde8b7c1 .........M.3....

*--> Trace <--*
Pc:00ac267b Fr:039cf284 Rt:00c4105e Arg:00000255 327882f8 039cf2a0 00c2cbb0 
Pc:00c4105e Fr:039cf294 Rt:00c2cbb0 Arg:000000b7 039cf2e8 00b68ab0 000000b7 
Pc:00c2cbb0 Fr:039cf2a0 Rt:00b68ab0 Arg:000000b7 00000000 327882f8 00000004 
Pc:00b68ab0 Fr:039cf2e8 Rt:624822b4 Arg:00000004 1fba9768 8bfda290 01d74750 
Pc:624822b4 Fr:039cfb78 Rt:62483af3 Arg:00000004 1fba9768 8bfda290 01d74750 
Pc:62483af3 Fr:039cfbcc Rt:62483944 Arg:00000004 1fba9768 00000000 00000001 
Pc:62483944 Fr:039cfbf0 Rt:62277520 Arg:00000004 1fbaa3c8 00000000 00000001 
Pc:62277520 Fr:039cfc2c Rt:6247bb58 Arg:039cfc7c 00c29544 039cfc50 039cfce0 
Pc:6247bb58 Fr:039cfc34 Rt:00c29544 Arg:039cfc50 039cfce0 009a6df0 009a7098 
Pc:00c29544 Fr:039cfc7c Rt:00c258ad Arg:05911b90 009a6e24 00000004 039cfce0 
Pc:00c258ad Fr:039cfd04 Rt:00c279bd Arg:009a6df0 0000d775 6fe419bf ffffffff 
Pc:00c279bd Fr:039cfd68 Rt:00abee01 Arg:00c277d0 ffffffff 7691e860 00000001 
Pc:00abee01 Fr:039cfd8c Rt:00ac1fe8 Arg:0000d775 00000002 00958dc0 00956238 
Pc:00ac1fe8 Fr:039cfdcc Rt:00abe528 Arg:00000002 00958dc0 00abe500 00958dc0 
Pc:00abe528 Fr:039cfde4 Rt:00bd90bb Arg:00956238 6fe41af7 00bd9064 00bd9064 
Pc:00bd90bb Fr:039cfe20 Rt:7691fa29 Arg:00958dc0 7691fa10 039cfe8c 77d37a4e 
Pc:7691fa29 Fr:039cfe30 Rt:77d37a4e Arg:00958dc0 8303b6db 00000000 00000000 
Pc:77d37a4e Fr:039cfe8c Rt:77d37a1e Arg:ffffffff 77d58907 00000000 00000000 
Pc:77d37a1e Fr:039cfe9c Rt:00000000 Arg:00bd9064 00958dc0 00000000 00000000 

*--> Stack <--*
039CF268  00fe10bc 00fe0f24 00fe0f24 00ac267b ....$...$...{&..
039CF278  039cf268 039cf284 6fe41653 039cf294 h.......S..o....
039CF288  00c4105e 00000255 327882f8 039cf2a0 ^...U.....x2....
039CF298  00c2cbb0 000000b7 039cf2e8 00b68ab0 ................
039CF2A8  000000b7 00000000 327882f8 00000004 ..........x2....
039CF2B8  8bfda290 01d74750 00000000 039cf2e8 ....PG..........
039CF2C8  622e336b 3279540c 039cf320 039cf324 k3.b.Ty2 ...$...
039CF2D8  039cf328 039cf318 039cf32c 327953f8 (.......,....Sy2
039CF2E8  039cfb78 624822b4 00000004 1fba9768 x...."Hb....h...
039CF2F8  8bfda290 01d74750 00000000 8bfda290 ....PG..........
039CF308  01d74750 1fbaa3c8 77d4ad10 f7428b8f PG.........w..B.
039CF318  8bfda290 01d74750 039cf330 00000004 ....PG..0.......
039CF328  039cf340 00000000 00000000 00000002 @...............
039CF338  19c4e0a0 00000000 1fba9768 039cf3d0 ........h.......
039CF348  77d1621f 0000005b 00000020 000000e0 .b.w[... .......
039CF358  00940000 00000003 00aae02d 00000002 ........-.......
039CF368  039cf388 000000ac 00aae02d 000000e8 ........-.......
039CF378  008507d0 00000064 00000000 000000e8 ....d...........
039CF388  00000000 1fc9c7e0 2a9851f8 00000000 .........Q.*....
039CF398  00000000 00850000 00000002 00000000 ................
039CF3A8  0000001d 00850000 0085022c 00000000 ........,.......
039CF3B8  000000e0 77d57804 22d153b0 00940000 .....x.w.S."....
039CF3C8  00000019 039cf3fc 039cf3f0 5cbdb094 ...............\
039CF3D8  00000000 623a867d 00000018 01068770 ....}.:b....p...
039CF3E8  22617fc8 1fc9c928 22617fe0 039cfd58 ..a"(.....a"X...
039CF3F8  039cf428 039cf434 00c40d5a 22617fe4 (...4...Z.....a"
039CF408  00000000 22617fe4 00000002 00000000 ......a"........
039CF418  00000000 1fc9c928 00000029 00000008 ....(...).......
039CF428  039cf428 039cf421 22617fc8 039cf444 (...!.....a"D...
039CF438  00c41018 00000029 00000000 039cf47c ....).......|...
039CF448  039cf47c 6247925a 6272962c 6247d4ea |...Z.Gb,.rb..Gb
039CF458  00000029 23fe3df8 1f148088 039cf400 )....=.#........
039CF468  00000009 039cf464 10000007 00000029 ....d.......)...
039CF478  00000000 039cf4a0 00da3486 10000007 .........4......
039CF488  00000029 00000000 00000029 23fe3df8 ).......)....=.#
039CF498  0577d410 27a273a0 039cf4e8 00ddd391 ..w..s.'........
039CF4A8  00000029 3000000c 0000000c 1f148088 )......0........
039CF4B8  1f148084 622fe809 31258b90 2ac2a564 ....../b..%1d..*
039CF4C8  3125839c 6270caf8 039cf4f4 622fcdd3 ..%1..pb....../b
039CF4D8  3000000c 625c9ee0 2ac2a560 23fe3df8 ...0..\b`..*.=.#
039CF4E8  039cf508 23fe3df8 039cf510 00dabe41 .....=.#....A...
039CF4F8  23fe3df8 23fe3df8 00db5b6d 00000000 .=.#.=.#m[......
039CF508  23fe3df8 00000008 039cf51c 00dba84a .=.#........J...
039CF518  0000001b 039cf52c 00da45d3 00000008 ....,....E......
039CF528  00000008 039cf570 00dd317d 00000029 ....p...}1..)...
039CF538  00000008 31258358 31258358 00dd319a ....X.%1X.%1.1..
039CF548  6227b797 22a83f98 31258b98 039cf588 ..'b.?."..%1....
039CF558  00000001 039cf564 00dbed14 039cf570 ....d.......p...
039CF568  00da34ac 31258384 31258358 3125839c .4....%1X.%1..%1
039CF578  039cf5a0 00da09c8 00000020 ffffffff ........ .......
039CF588  31258b9c 00000004 039cf5b0 31258358 ..%1........X.%1
039CF598  000005b4 01066290 039cfb74 00da1134 .....b..t...4...
039CF5A8  00000058 039cfb8c 0000c800 02000000 X...............
039CF5B8  18010000 81010005 ef1446b0 24d7c18f .........F.....$
039CF5C8  000a00ef 00000000 00200000 0000000a .......... .....
039CF5D8  20001389 7e000901 f000c6f4 00004595 ... ...~.....E..
039CF5E8  bf7fd229 3d192c83 90000001 80000043 )....,.=....C...
039CF5F8  2800003f 6f6e6342 0000006e 00000000 ?..(Bcnon.......
039CF608  00000000 000f3e00 00000000 00000000 .....>..........
039CF618  00000000 80000000 8000007f 0000007f ................
039CF628  00000000 007f8000 007f8000 0a006c00 .............l..
039CF638  00000000 00000000 58000000 00000500 ...........X....
039CF648  00001000 9294ab00 00000180 cccc0000 ................
039CF658  0009cccc 00610050 00610072 00570020 ....P.a.r.a. .W.

*--> Error Logs <--*
(2) Model: file '0x140e49' could not be opened (2)
(2) Texture '' file not found

*--> DirectX Device Info <--*
VendorId    = 0x10de
DeviceId    = 0x1c8d
Version     = 27.21.0014.6611
Description = NVIDIA GeForce GTX 1050
Compat      = 0x01000001
VidMem      = 4095 MB

*--> DirectSound Device Info <--*
3D channels  = 0
2D channels  = 0
EAX support  = None
Device count = 2
```

**Comment by kazerniel** [comment]
I also crashed twice today, once when trying to move from an explorable area to an outpost, second time when trying to use the Travel menu. Friend list module is not enabled.

**Comment by 3vcloud** [comment]
I'm going to bed now, please test the new version for me and post more problems here for me to enjoy my morning coffee to please

**Comment by BearsMan** [comment]
> I'm going to bed now, please test the new version for me and post more problems here for me to enjoy my morning coffee to please

Of course good night :)

**Comment by BearsMan** [comment]
Crash occurred on Version 5.1

[ArenaNet.log](https://github.com/HasKha/GWToolboxpp/files/6469549/ArenaNet.log)

[Crash.zip](https://github.com/HasKha/GWToolboxpp/files/6469551/Crash.zip)

**Comment by 3vcloud** [comment]
Ah ok, after all this crashing it turns out that the issue is with the observer module - will fix thanks

**Comment by BearsMan** [comment]
Ok the issue was fixed on my end via the observer module

**Comment by 3vcloud** [comment]
Fixed 5.3
